### PR TITLE
fix pnn patch for Oracle

### PR DESCRIPTION
**Pull request by ticoann**
@alexanderrichards, Alex please review.

### CODE REVIEW

File: src/python/WMCore/WMBS/Oracle/Locations/New.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class New(NewLocationMySQL):
                                WHERE site_name = :location)"""
 
     seSQL = """INSERT INTO wmbs_location_senames (location, se_name)
-                 SELECT wl.id, :se_name FROM wmbs_location wl
+                 SELECT wl.id, :pnn FROM wmbs_location wl
                  WHERE wl.site_name = :location
                  AND NOT EXISTS (SELECT null FROM wmbs_location_senames wls2 WHERE
                                   wls2.se_name = :se_name
```
Comment (hufnagel) [inline src/python/WMCore/WMBS/Oracle/Locations/New.py]: Missed changing that se_name to pnn.

With this it now at least deploys, lets see if it works...

**Comment by alexanderrichards** [comment]
@ticoann looks good to me apart from the missing se_name -> pnn that Dirk spotted and commented on above

File: src/python/WMCore/WMBS/Oracle/Locations/New.py (hunk 1 above)
Comment (ticoann) [inline src/python/WMCore/WMBS/Oracle/Locations/New.py]: Thanks Dirk, Alex, I updated that.

**Comment by amaltaro** [comment]
btw, I also ran a few workflows that already completed and they look Ok. Logs are also normal, no errors or crashes.

**Pull request merged**